scripts/omg.yml

```
@@ -29,4 +29,5 @@ runcmd:
     EOF
   - curl iiab.io/risky.txt | bash &
   - curl https://raw.githubusercontent.com/iiab/calibre-web/master/scripts/calibre-web-nginx.conf > /etc/nginx/conf.d/calibre-web-nginx.conf
+  - ln -s /library/calibre-web /library/wwww/html/calibre-web
```

Side Question: Does this allow everyone access to all books, regardless whether they have a Calibre-Web account/access/authorization?

(As an intentional or unintentional side effect?)

```suggestion
  - ln -s /library/calibre-web /library/www/html/calibre-web
```

---

Unfortunately, yes. I will consider this as a temporary fix. This should be allowed only within a Calibre-Web session/context. A guest user doesn't need authorization to play a specific video, but should not have access to actual directory of all files. I need some help to properly design a more secure solution for this.